SystemConnectors/WindowsConnector/MediaController.py

```diff
@@ -0,0 +1,41 @@
+"""
+Media controller for Windows
+"""
+import win32api
+
+class MediaController(object):
+    """
+    Keycodes to send to the operating system for corresponding action
+    """
+    PLAY_PAUSE_KEYCODE = 0xB3
+    SKIP_KEYCODE = 0xB0
+    PREV_KEYCODE = 0xB1
+
+    """
+    Helper function to send the key press to Windows
+    """
```

Comment:
Python be funky language where function docstring goes inside the function. I don't feel like commenting this on all functions